stage0/layout.ld

```diff
@@ -107,6 +107,11 @@ SECTIONS {
     } > ram_low
     ASSERT(SIZEOF(.boot.cpuid) == 4K, "CPUID section has to be exactly one page in size")
 
+    .boot.ghcb (NOLOAD) : {
```

Comment:
Not sure about the linker script format, but should this also specify an alignment to make sure it is page-aligned?

Reply:
It shouldn't matter as the GHCB struct itself is `align(4096)`, but it won't hurt to add it here as well. (The resulting BIOS blob is unchanged.)